hack/lib/util/golang.sh

```diff
@@ -8,10 +8,10 @@ function os::golang::verify_go_version() {
 
 	local go_version
 	go_version=($(go version))
-	if [[ "${go_version[2]}" != go1.7* ]]; then
+	if [[ "${go_version[2]}" != go1.8* ]]; then
 		os::log::info "Detected go version: ${go_version[*]}."
 		if [[ -z "${PERMISSIVE_GO:-}" ]]; then
-			os::log::error "Please install Go version 1.7 or use PERMISSIVE_GO=y to bypass this check."
+			os::log::error "Please install Go version 1.8 or use PERMISSIVE_GO=y to bypass this check."
```

Comment:
This needs to be `os::log::fatal` -- the conversation earlier was re: the fact that David made _this_ check a no-op in the reabse (see the `return 0` below).